tests/page/locator-misc-2.spec.ts

```diff
@@ -82,3 +82,12 @@ it('should return bounding box', async ({ page, server, browserName, headless, i
   const box = await element.boundingBox();
   expect(box).toEqual({ x: 100, y: 50, width: 50, height: 50 });
 });
+
+it('should waitFor', async ({ page }) => {
```

Comment:
Let's add a test with `waitFor({ hidden })` or some other state passed, otherwise we lack coverage.